resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/v1/Client.scala

```diff
@@ -152,6 +152,7 @@ private[spark] class Client(
           .pods()
           .withName(kubernetesAppId)
           .watch(loggingWatch)) { _ =>
+        loggingWatch.start
```

Comment:
nit: methods with side effects that have no parameters usually get the parentheses: `.start()`